### PR TITLE
Coda IO

### PR DESCRIPTION
**Pull request by as2388**
Supports exporting to Coda and applying codes from a coded Coda data-file

### CODE REVIEW

**Comment by as2388** [comment]
Please review #10 first

File: tests/traced_data/resources/coda_export_expected_output_coded.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+id;owner;data;timestamp;schemeId;schemeName;deco_codeValue;deco_codeId;deco_confidence;deco_manual;deco_timestamp;deco_author
+0;0;female;;;;;;;;;
+1;1;m;;;;;;;;;
+2;2;WoMaN;;;;;;;;;
+3;3;27;;;;;;;;;
```
Comment (lukechurch) [inline tests/traced_data/resources/coda_export_expected_output_coded.csv]: I don't remember what we discussed about this - could we ensure these all have a trailing new line at the end of the lines?

File: tests/traced_data/resources/coda_import_data.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+id;timestamp;owner;data;schemeId;schemeName;deco_codeValue;deco_codeId;deco_confidence;deco_manual;deco_timestamp;deco_author
+6469567482;;3885485969;female;1;CodaCodedGender;F;1-48;0.95;true;Fri May 18 2018 12:26:14 GMT+0100 (BST);
+4597828651;;9051968373;m;1;CodaCodedGender;M;1-01;0.95;true;Fri May 18 2018 12:26:13 GMT+0100 (BST);
+1020892817;;2494941557;WoMaN;1;CodaCodedGender;F;1-48;0.95;true;Fri May 18 2018 12:26:12 GMT+0100 (BST);
+8083248450;;7380450267;27;1;CodaCodedGender;;;;;;
```
Comment (lukechurch) [inline tests/traced_data/resources/coda_import_data.txt]: Why is this a .txt file, not a .csv?

File: tests/traced_data/resources/coda_export_expected_output_coded.csv (hunk 1 above)
Comment (as2388) [reply tests/traced_data/resources/coda_export_expected_output_coded.csv]: Not without breaking Coda. There is an open issue in Coda for this (https://github.com/AfricasVoices/coda/issues/137), but I didn't think fixing it was an immediate priority.

File: tests/traced_data/resources/coda_import_data.txt (hunk 2 above)
Comment (as2388) [reply tests/traced_data/resources/coda_import_data.txt]: Because that's what Coda's export function gives you...

Shall I make an issue for this too?

**Comment by as2388** [comment]
Will merge now, with an issue (#12) reminding us to update test files once Coda is fixed.

**Pull request merged**